pkgs/development/compilers/miranda/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, gcc }:
+
+stdenv.mkDerivation rec {
+  version = "2.066";
+  pname = "miranda";
+
+  src = fetchurl {
+    url = "https://www.cs.kent.ac.uk/people/staff/dat/miranda/src/mira-${builtins.replaceStrings [ "." ] [ "" ] version}-src.tgz";
+    sha256 = "1sqn031agwwgq3pka55pxd6m22dagmkxjhbwrni8gpigihg2h72j";
+  };
+
+  dontStrip = true;
+
+  enableParallelBuilding = true;
+
+  configurePhase = ''
+    patchShebangs quotehostinfo
+  '';
+
+  makeFlags = [ "CFLAGS=-O2" ];
+
+  buildFlags = [ "cleanup" "all" ];
+
+  installPhase = ''
+    mkdir -p $out/bin
```

Comment:
When overriding `installPhase`, you should also provide `runHook preInstall` and `runHook postInstall`.

Though, in this case, it might be sufficient to rely on the default `installPhase` when setting:

```
makeFlags = [
  "BIN=${placeholder "out"}/bin"
  "LIB=${placeholder "out"}/lib"
  "MAN=${placeholder "out"}/share/man/man1"
];
```

Reply:
Doing so results in http://ix.io/2Age

Reply:
Looks like we would also need to create those directories in `preInstall`.